securing_apps/topics/oidc/nodejs-adapter.adoc

```diff
@@ -80,6 +80,17 @@ object, rather than the `keycloak.json` file:
     let keycloak = new Keycloak({ store: memoryStore }, kcConfig);
 ----
 
+Behind a proxy::
+By specifying the attribute `proxyUrl` property in the configuration object, you can yse this adapter behind a proxy server.
```

Comment:
You have a typo here "yse". I'd appreciate if you fix it.

Reply:
Oh, sorry! Of course!